frontend/src/context/ws-client-provider.tsx

```diff
@@ -5,6 +5,8 @@ import ActionType from "#/types/ActionType";
 import EventLogger from "#/utils/event-logger";
 import AgentState from "#/types/AgentState";
 
+const RECONNECT_RETRIES = 5;
```

Comment:
I think maybe we should retry in perpetuity?

E.g. maybe your wifi goes down for 5m on a flight--it'd be nice if OpenHands just perked back up after the wifi came back on